packages/bsky/src/data-plane/server/indexing/plugins/post.ts

```diff
@@ -334,6 +367,23 @@ const deleteFn = async (
   }
   if (deletedPosts) {
     deletedEmbeds.push(deletedPosts)
+
+    await db.deleteFrom('quote').where('uri', '=', uriStr).execute()
+    await db
+      .insertInto('post_agg')
+      .values({
+        uri: deletedPosts.embedUri,
+        quoteCount: db
+          .selectFrom('quote')
+          .where('quote.subjectCid', '=', deletedPosts.embedCid)
+          .select(countAll.as('count')),
+      })
+      .onConflict((oc) =>
+        oc
+          .column('uri')
+          .doUpdateSet({ quoteCount: excluded(db, 'quoteCount') }),
+      )
+      .execute()
```

Comment:
Flip side of the insertion path— maybe we only need to do this when the embed is a post.

Reply:
Same question here, maybe I'm wrong but I assumed this would only be posts.

```tsx
    db
      .deleteFrom('post_embed_record')
      .where('postUri', '=', uriStr)
      .returningAll()
      .executeTakeFirst(),
```

Reply:
I believe this deletes all embeds for the post identified by `uriStr `.  But the embedded record isn't necessarily a post (e.g. could be a list, feedgen, or starter pack).  And in those cases we don't consider them QPs.

Reply:
Oh, interesting! That makes sense. What is the way to check this then? Checking the `uri`?

Reply:
Think I see how we did it elsewhere!